### PR TITLE
initial /me/drives implementation

### PR DESCRIPTION
**Pull request by butonic**
Inital spaces API, following https://github.com/owncloud/ocis/pull/1827
requires https://github.com/cs3org/reva/pull/1678

### CODE REVIEW

**Comment by butonic** [comment]
cc @dragotin

**Comment by butonic** [comment]
blocked by reva PR https://github.com/cs3org/reva/pull/1802

File: graph/pkg/service/v0/drives.go [hunk 1]
```diff
@@ -139,3 +189,73 @@ func formatDriveItems(mds []*storageprovider.ResourceInfo) ([]*msgraph.DriveItem
...
+		webDavURL := baseURL.String() + rootID
+		drive.Root.WebDavUrl = &webDavURL
+	}
+
+	if space.Mtime != nil {
+		lastModified := cs3TimestampToTime(space.Mtime)
+		drive.BaseItem.LastModifiedDateTime = &lastModified
+	}
+	if space.Quota != nil {
+		// FIXME use https://github.com/owncloud/open-graph-api and return proper int64
```
Comment (micbar) [inline graph/pkg/service/v0/drives.go:234]: remove comment?

File: graph/pkg/service/v0/drives.go [hunk 2]
```diff
@@ -95,34 +143,36 @@ func (g Graph) GetRootDriveChildren(w http.ResponseWriter, r *http.Request) {
...
 		},
 		Size: size,
 	}
+	if res.Mtime != nil {
+		lastModified := cs3TimestampToTime(res.Mtime)
+		driveItem.BaseItem.LastModifiedDateTime = &lastModified
+	}
 	if res.Type == storageprovider.ResourceType_RESOURCE_TYPE_FILE {
-		driveItem.File = &msgraph.File{
+		driveItem.File = &msgraph.OpenGraphFile{
```
Comment (micbar) [inline graph/pkg/service/v0/drives.go:170]: Comment why weird type?

**Review by micbar — APPROVED** [review]
only cosmetic stuff

**Pull request merged**